test/core/invariant/handlers/LockupHandler.sol

```diff
@@ -167,7 +167,7 @@ contract LockupHandler is BaseHandler {
         }
 
         // Withdraw from the stream.
-        lockup.withdraw({ streamId: currentStreamId, to: to, amount: withdrawAmount });
+        lockup.withdraw{ value: SABLIER_FEE }({ streamId: currentStreamId, to: to, amount: withdrawAmount });
```

Comment:
Why do we need a non-zero `SABLIER_FEE` in invariant if fee is not a part of any invariant statement?

Reply:
Good point, we don't need it